### PR TITLE
Add basic doc for SQL analyist

### PR DESCRIPTION
**Pull request by Leemoonsoo**
Add basic JDBC doc. this branch is based on #18.

JDBC page is located under Interpreter menu and looks like
image

Added "Secure JDBC connection" section
image

### CODE REVIEW

**Comment by AhyoungRyu** [comment]
It seems there are two unrelated commits. Only "Add jdbc interpreter doc" is necessary in here, right? 
screen shot 2017-10-18 at 3 05 01 pm

**Comment by Leemoonsoo** [comment]
Yes, this branch is on top of #18.

**Comment by Leemoonsoo** [comment]
Rebased after merge #18

File: docs/guide/interpreter/jdbc.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  - Database is currently up and running
+  - Database is accessible from public internet
+  - You have proper credentials to access database
+
+## Create new JDBC Interpreter
+
+First, you'll need to create an intepreter to provide database connection information.
+
+1. Click **Create interpreter** button from [Interpreter](https://www.zepl.com/settings/interpreters) page.
+2. Select **jdbc** in Interprer group
```
Comment (AhyoungRyu) [inline docs/guide/interpreter/jdbc.md]: A typo : `Interprer` -> `Interpreter`

File: docs/guide/interpreter/jdbc.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+```
+%psql
+SELECT * from my_table
+```
+
+## Secure database connection
+
+### Whitelist IP addresses
+ZEPL connects your database using two IP addresses **52.38.31.66** and **52.32.218.160**. It's always good idea setup firewall and whitelist the IP.
+
```
Comment (AhyoungRyu) [inline docs/guide/interpreter/jdbc.md]: Maybe "to set firewall and whitelist the IP up" ? @rbpark i need your confirm ㅋㅋㅋ

**Review by AhyoungRyu — APPROVED** [review]
Except a typo and grammar error, Looks Good To Me

File: docs/guide/interpreter/jdbc.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+<h1> JDBC Interpreter </h1>
+
+ZEPL supports JDBC interpreter with drivers for popular databases. To connect your database, it's important to check
```
Comment (rbpark) [inline docs/guide/interpreter/jdbc.md]: Maybe: "Before connecting to your database..." is better than "To connect..."? It's not important, but it feels more natural.

Also "it's important to check the following:"

File: docs/guide/interpreter/jdbc.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+  - Database is accessible from public internet
+  - You have proper credentials to access database
+
+## Create new JDBC Interpreter
+
+First, you'll need to create an intepreter to provide database connection information.
+
+1. Click **Create interpreter** button from [Interpreter](https://www.zepl.com/settings/interpreters) page.
+2. Select **jdbc** in Interprer group
+3. Type **Interpreter name**. This name will be used in the notebook to call this JDBC interpreter.
```
Comment (rbpark) [inline docs/guide/interpreter/jdbc.md]: Maybe `Provide an **Interpreter name**`

File: docs/guide/interpreter/jdbc.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+4. Type **JDBC connection URL**, **username** and **password**.
```
Comment (rbpark) [inline docs/guide/interpreter/jdbc.md]: Provide I think is better here too.

File: docs/guide/interpreter/jdbc.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+5. Select **JDBC driver**
```
Comment (rbpark) [inline docs/guide/interpreter/jdbc.md]: `Select a **JDBC driver**`

File: docs/guide/interpreter/jdbc.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+You can use **Test connection** button to test connection.
```
Comment (rbpark) [inline docs/guide/interpreter/jdbc.md]: Use the **Test connection** button to test the connection.

File: docs/guide/interpreter/jdbc.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+### Use your JDBC Interpreter
+
+Once you have created JDBC interpreter, you can call it in the notebook using `%[Interpreter name]` directive. For example if you have created your JDBC interpreter with name "psql", you can call it `%psql` in the notebook. i.e.
```
Comment (rbpark) [inline docs/guide/interpreter/jdbc.md]: Once you have created **the** JDBC interpreter, you can use it in your notebook by providing the `%[Interpreter name]` directive.  For example **,** if you have created your JDBC interpreter with name "psql", you can use `%psql` in the notebook. i.e.

File: docs/guide/interpreter/jdbc.md (hunk 2 above)
Comment (rbpark) [reply docs/guide/interpreter/jdbc.md]: @AhyoungRyu no :( Nice try, but failure.
Moon was close:
`It's always a good idea to setup a firewall and whitelist external IPs.`

Or `It's recommended to ...`

**Review by rbpark — COMMENTED** [review]
After the changes, it's good.

**Comment by Leemoonsoo** [comment]
Thanks @AhyoungRyu @rbpark for review. Addressed all comments.

**Pull request merged**